.gitignore

```diff
@@ -22,3 +22,7 @@ cscope.out
 /cmus.base
 /cmus.def
 /cmus.exp
+GPATH
+GRTAGS
+GSYMS
+GTAGS
```

Comment:
Not related to the pl-flush/reload. Also specific to your workflow.

Reply:
Removed